docs/prover/cli/conf-file-api.md

```diff
@@ -0,0 +1,102 @@
+Certora Prover Conf File API
+============================
+
+The **Conf File API** is an alternative API for  `certoraRun` . In terms of functionality 
+this API is identical to the [CLI Options](options.md) API. Instead of calling `certoraRun` 
+with a list of shell flags, some or all the flags can be stored in a [JSON](https://www.json.org/json-en.html) file 
+(to be more precise the format is [JSON5](https://json5.org/)):
+
+```
+certoraRun my_params.conf
+```
+
+Conf files must use the <strong>.conf</strong> suffix
+
+
+How CLI Options are mapped to JSON
+----------------------------------
+
+**JSON Keys**
```

Comment:
Use the heading commands (e.g. `### JSON Keys`) rather than bolding to create headers.

Reply:
Although in this case, I would probably skip the headers in favor of a bulleted list.  E.g, something like
> Command-line arguments are stored as key-value pairs in the conf file.  The keys are the names of the parameters (with the leading `--` removed).  For example,
> ```{sh}
> certoraRun --verify Example:example.spec
> ```
> is equivalent to running with the following conf file:
> ```{json}
> { "verify": "Example:example.spec" }
> ```
> 
> The values in the map depend on the type of arguments:
>  - for flags that take no additional arguments (such as {ref}`--send_only`), the value should be `true`.  For example,
>    ```{sh}
>    certoraRun --send_only
>    ```
>    would be encoded as:
>    ```{json}
>    { "send_only": true }
>    ```
>  
>  - flags that take a single additional argument (such as {ref}`--solc` or {ref}`--smt_timeout`) are encoded as a JSON string.  For example,
>    ```{sh}
>    certoraRun --solc solc4.25 --smt_timeout 600
>    ```
>    would be encoded as:
>    ```{json}
>    { "solc": "solc4.25", "smt_timeout": "600" }
>    ```
>    Note that conf files do not use JSON numbers; numbers are encoded as strings.
>  
>   - flags that take multiple additional arguments (such as {ref}`--packages`) are encoded as JSON lists.  For example,
>     ```{sh}
>     certoraRun --packages @balancer-labs/v2-solidity-utils=pkg/solidity-utils @balancer-labs/v2-vault=pkg/vault"
>     ```
>     would be encoded as:
>     ```{json}
>     {
>         "packages": [
>             "@balancer-labs/v2-solidity-utils=pkg/solidity-utils",
>             "@balancer-labs/v2-vault=pkg/vault"
>         ]
>     }
>     ```